pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1290,6 +1285,9 @@ pub mod pallet {
 						affiliate_config.affiliator_enable_fee,
```

Comment:
missing the check if it is already an Affiliator we don't want to let them pay two times

Reply:
Added checks!